includes/admin/class-give-settings.php

```diff
@@ -382,7 +382,7 @@ public function give_settings( $active_tab ) {
 						),
 						array(
 							'name' => esc_html__( 'Collect Billing Details', 'give' ),
-							'desc' => esc_html__( 'This option will enable the billing details section for offline donations. The fieldset will appear above the offline donation instructions. Note: You may customize this option per form as needed.', 'give' ),
+							'desc' => esc_html__( 'Enable billing details section for offline donations. The fieldset will appear above the offline donation instructions. Note: You may customize this option per form as needed.', 'give' ),

```

Comment:
Correction: "Enable to request billing details for offline donations. Will appear above offline donation instructions. Can be enabled/disabled per form."